templates/case-studies/templates/_macros-quote.jinja

```diff
@@ -0,0 +1,28 @@
+# Params
+# quote: Quote text (required)
+# name: Name of quoter (required)
+# job_title: Job title of quoter (required)
+
+{%- macro quote(quote, name, job_title) -%}
+  <section class="p-strip">
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <hr />
+        <div class="row">
+          <div class="col-6">
+            <blockquote class="p-pull-quote" style="margin: 0;">
```

Comment:
```suggestion
            <blockquote class="p-pull-quote" style="margin: 0;">
```
is this `style="margin: 0;"` something that could be put in a class?

Reply:
I created a class for p-pull-quote for this style and also for the quotes colour